mongodbatlas/global_clusters_test.go

```diff
@@ -31,7 +31,7 @@ func TestGlobalClusters_Get(t *testing.T) {
 	groupID := "1"
 	clusterName := "appData"
 
-	mux.HandleFunc(fmt.Sprintf("/api/atlas/v1.0/groups/%s/clusters/%s/globalWrites", groupID, clusterName), func(w http.ResponseWriter, r *http.Request) {
+	mux.HandleFunc(fmt.Sprintf("/api/atlas/v1.5/groups/%s/clusters/%s/globalWrites", groupID, clusterName), func(w http.ResponseWriter, r *http.Request) {
```

Comment:
One question, not to @Sugar-pack probably. This path is quite similar to the one defined by the `globalClustersBasePath` constant. Maybe it would be better to use this constant here and in other places.

Reply:
cc @andreaangiolillo